common/logic.proto

```diff
@@ -114,9 +115,13 @@ message Explanation {
     Rule rule = 1;
     map<string, VarList> var_mapping = 2;
     ConceptMap condition = 3;
-    ConceptMap conclusion = 4;
+    ConclusionAnswer conclusion = 4;
 
     message VarList {
         repeated string vars = 1;
     }
+
+    message ConclusionAnswer {
+        map<string, Concept> map = 1;
+    }
```

Comment:
This message/object is not meaningful given that it wraps one object -- it's a thin wrapper and not canonical. We should keep a simple <map, Concept> message type for conclusion field in Explanation message type.